.circleci/owasp/jobs.yml

```diff
@@ -42,7 +42,7 @@
           target: frontend
 
   nightly-owasp-scan:
-    executor: machine-executor
+    executor: zap-machine
```

Comment:
@ADPennington We modify all nightly scans to use this larger resource including the HHS relevant branches noted below.